media/css/m24/hero.scss

```diff
@@ -3,31 +3,123 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 
-.m24-c-hero {
-    @include container;
-    padding-top: $spacer-2xl;
+#about .m24-c-hero {
+    padding-top: $spacer-lg;
 
-    * {
-        margin-bottom: 0;
+    .m24-c-copy {
+        padding: $spacing-2xl $spacer-md;
+        max-width: none;
+
+        .m24-c-copy-wrapper {
+            max-width: $content-max;
+            margin: auto;
+        }
     }
-}
 
-.m24-c-hero-title {
-    font-size: $text-title-2xl;
+    .m24-c-hero-title {
+        font-size: $text-title-2xl;
+    }
 }
 
 .m24-c-hero-media {
-    margin: 0 calc(-1 * $container-padding);
-    padding-top: $spacer-xl;
+    margin: 0;
+    background-color: $color-black;
+}
+
+.m24-c-hero-media-container {
+    max-width: $content-max;
+    margin: auto;
+    padding: $spacer-md $spacer-md 0;
+
+    @media #{$mq-md} {
+        padding: $spacer-lg $spacer-md 0;
+    }
 
     img {
         display: block;
-        border: $spacer-xs solid $color-black; // this does not change from light to dark mode
         box-sizing: border-box;
+        width: 100%;
+        margin: 0 auto;
+    }
+
+    .m24-c-hero-media-content {
+        padding: $spacer-xl 0;
+    }
+
+    .m24-c-hero-media-heading {
+        color: $color-white;
+        font-size: $text-title-xl;
+        line-height: 0.95;
+        margin-bottom: $spacer-xl;
+
+        @media #{$mq-md} {
+            width: 75%;
+        }
+
+        @media #{$mq-lg} {
+            width: 60%;
+        }
+    }
+
+    .m24-c-hero-media-cta {
+        padding: 0;
+        text-align: right;
```

Comment:
**Suggestion:** should this use `bidi()` for RTL styles? It might not be needed now but if it's easy to add it's something we don't have to follow up and fix later on.

Reply:
Updated to use `text-align: end` instead